echopype/consolidate/split_beam_angle.py

```diff
@@ -0,0 +1,574 @@
+"""
+Contains functions necessary to compute the split-beam (alongship/athwartship)
+angles and add them to a Dataset.
+"""
+from typing import List, Optional, Tuple
+
+import numpy as np
+import xarray as xr
+
+from ..echodata import EchoData
+
+
+def _get_splitbeam_angle_power_CW(ds_beam: xr.Dataset) -> Tuple[xr.Dataset, xr.Dataset]:
+    """
+    Obtains the split-beam angle data from power encoded data with CW waveform.
+
+    Parameters
+    ----------
+    ds_beam: xr.Dataset
+        An ``EchoData`` beam group containing angle information needed for
+        split-beam angle calculation
+
+    Returns
+    -------
+    theta_fc: xr.Dataset
+        The calculated split-beam alongship angle
+    phi_fc: xr.Dataset
+        The calculated split-beam athwartship angle
+
+    Raises
+    ------
+    NotImplementedError
+        If all ``beam_type`` values are not equal to 1
+
+    Notes
+    -----
+    Can be used on both EK60 and EK80 data
+
+    Computation done for ``beam_type=1``:
+    ``physical_angle = ((raw_angle * 180 / 128) / sensitivity) - offset``
+    """
+
+    # raw_angle scaling constant
+    conversion_const = 180.0 / 128.0
+
+    # TODO: is this function useful or just annoying? Should we put it outside this function?
+    def compute_split_beam_beamtype1(angle_type: str) -> xr.Dataset:
+        """Compute a split-beam angle for ``beam_type=1`` based on ``angle_type``"""
```

Comment:
I think this can just sit inside here as a private function. Some suggestions below
```suggestion
    # TODO: is this function useful or just annoying? Should we put it outside this function?
    def _e2f(angle_type: str) -> xr.Dataset:
        """Convert electric angle to physical angle for split-beam data"""
```